savaslabs/sumac
# Issue #1675 report failed time entries

@chrisarusso assigning to you for a once-over, feel free to pass along if you think someone else is better-equipped. Once this is merged the docker hub image will automatically rebuild and we'll have to touch base about updating the cron job you have to run docker instead of PHP.

To test:
* Spin up [Redmine](https://github.com/savaslabs/redmine_savas) locally using Docker and the instructions in the readme
* copy config.example.yml to config.yml
* In Config.yml:
* Set `debug-user` under `slack` to your slack name (e.g. `@tim`)
* Set the redmine URL to `http://local.pm.savaslabs.com`
* Log into your local redmine instance (same username/password as production redmine), click on "My Account", click "show" under "API Access Key" and copy that API Access Key into the Redmine API Access Key in config.yml
* Get the slack webhook URL from @timstallmann and fill that into config.yml
* Run `docker build -t savaslabs/sumac:test .` in the repo directory to build the docker image
* Use the run command examples in the readme to run sync, but add the `:test` tag to the docker image name to make sure you're using your local image.
